### ISSUE TITLE
Updates

### ISSUE THREAD

**Issue by bittermandel** [mbrochh/django-react-primer#2]
Hey! I read through the whole thing and noticed it is a tiny bit outdated. Are you planning to re-write it? Or are you allowing pull requests?

Thanks :)

**Comment by mbrochh** [comment]
Indeed! I have actually JUST had my little breakthrough moment where I am finally able to use ReactJS with Django in the real world and I documented it here: https://github.com/mbrochh/django-reactjs-boilerplate

I should definitely update this primer (i.e.I settled on Radium for inline styles), Redux for state management, isomorphic-fetch for API calls.

The only thing that I still haven't figured out is universal apps (mainly because I am reluctant to run a Node.js "render-farm" next to my Django webservers).

I would accept pull requests, but bear in mind that this is an opinion piece and I'd feel very bad if you put a lot of time into writing a pull request but then I don't like it because you use libraries that I'm not familiar with. What changes do you have in mind specifically?
